src/main/java/com/microsoft/lst_bench/util/StringUtils.java

```diff
@@ -75,4 +83,20 @@ public static FileExec replaceRegex(FileExec f, String regex, String replacement
                         s.getId(), pattern.matcher(s.getStatement()).replaceAll(replacement)))
             .collect(Collectors.toList()));
   }
+
+  /**
+   * Reads the contents of the `sourceFile` and replaces any environment variables if present. If
+   * the environment variable is not set, the default value is used if specified. All other
+   * parameters are ignored.
+   */
+  public static String replaceEnvVars(File sourceFile) throws IOException {
+    if (sourceFile == null || !sourceFile.isFile()) {
+      // Nothing to do.
+      LOGGER.debug("replaceEnvVars received a null or missing file.");
+      return null;
+    }
+    StringSubstitutor envSub = new StringSubstitutor(System.getenv());
+    envSub.setValueDelimiter(":");
```

Comment:
Any reason to use ':' rather than default ':-'?

Reply:
Done, I was previously referring to an older version that used ":" as the delimiter. I have updated it to use ":-".